### ISSUE TITLE
How can i integrate this with my PHP website?

### ISSUE THREAD

**Issue by usmancs08** [Casualtek/Cyberwatch#1]
Hi I have a website i want to integrate Cyberwatch with it can you help me with that? Is there any API's for this?

**Comment by Casualtek** [comment]
What do you want to integrate?
PHP can process JSON files...

**Comment by usmancs08** [comment]
Yes it can process JSON files how can i get news feeds for cyber attacks from this script on daily bases?

**Comment by usmancs08** [comment]
Ok i think i got the point i can get the json data which updated regularly but how can i get this data in another language in my case it would be german?

**Comment by Casualtek** [comment]
Well, if I were you, I'd make a python script that grabs the raw JSON file, takes the summary of each new item and translates it into German using DeepL's API (or even the API of a generative AI). I'd have it run with GitHub actions daily to generate what you want for your PHP file to process.

**Comment by Casualtek** [comment]
I'm generating a HTML weekly digest in German for my colleagues. But APIs are not all free and I won't use any to either produce a German or English version of the JSON file.

**Comment by usmancs08** [comment]
Got you thanks for you helps it meant alot thanks again
